src/app/organizer/Columns.tsx

```diff
@@ -109,7 +109,7 @@ export function getColumns(
   };
 
   // Some stat labels are long. This lets us replace them with i18n
-  const statLabels = {
+  const statLabels: Record<number, string> = {
```

Comment:
`const statLabels: Record<number, string|undefined> = {`
as long as we are typing explicitly

Reply:
Record implies the `undefined` part I think.

Reply:
Huh I am totally wrong